accounts/abi/bind/backends/simulated.go

```diff
@@ -117,7 +117,10 @@ func (b *SimulatedBackend) ContractCall(contract common.Address, data []byte, pe
 		data:     data,
 	}
 	// Execute the call and return
-	vmenv := core.NewEnv(statedb, core.TestConfig, b.blockchain, msg, block.Header())
+	vmenv, err := core.NewEnv(statedb, core.TestConfig, b.blockchain, msg, block.Header())
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Hmm, just want to get some idea of what happened here. Originally `NewEnv` doesn't return any error, what makes it changed?

Reply:
VM can be started as separate process or even on an other host. So there can be some problems with start/connect VM.

Reply:
I'm trying to be more careful about what actually happened when there's problem with connecting/starting the VM. After all, the client cannot process blocks or validate new transactions if the VM is not there. Thus when the VM fails, I think there's two logical ways for the client to handle it -- either a) pause everything it's currently doing and re-connect until successful, or b) panic and exit. Either way, I suggest us to be more careful when dealing with those early returns.

In the mean time, probably the client should just client to the VM just once in bootstrap and use that session for all future VM executions?

Reply:
Any idea/plan to fix this?